cmd/microcloud/preseed.go

```diff
@@ -364,6 +460,77 @@ func (p *Preseed) validate(name string, bootstrap bool) error {
 	return nil
 }
 
+// isInitiator returns true if the current host is marked as being the initiator.
+func (p *Preseed) isInitiator(name string, address string) bool {
+	if name == p.Initiator || address == p.InitiatorAddress {
+		return true
+	}
+
+	return false
+}
+
+// initiatorAddress returns the address of the trust establishment session's initiator.
+// This is either the specified address of the initiator or an empty string.
+func (p *Preseed) initiatorAddress() string {
+	if p.InitiatorAddress != "" {
+		return p.InitiatorAddress
+	}
+
+	return ""
+}
```

Comment:
So looks like we don't infer the first system in the list is the initiator anymore for bootstrap? It's a required key?

I think we should error out if there are ambiguities like `name != "" && name != p.Initiator && address == p.InitiatorAddress`

Also that `initiatorAddress` helper doesn't seem to be doing much.

Reply:
Yeah this way in all scenarios (interactive, preseed) and modes (unicast, multicast) you have to either provide `initiator` or `initiator_address`. So it looks a bit more consistent.

The preseed's `validate()` already has a check in case either `initiator` or `initiator_address` are empty. 
I have added an additional check that doesn't allow setting both `initiator` and `initiator_address` at the same time.
Therefore things like `name != "" && name != p.Initiator && address == p.InitiatorAddress` shouldn't be anymore possible.